gnomad_qc/v3/create_release/create_hgdp_tgp_subset.py

```diff
@@ -0,0 +1,821 @@
+        global_annotation_descriptions=hl.literal(GLOBAL_SAMPLE_ANNOTATION_DICT),
+        sample_annotation_descriptions=hl.literal(SAMPLE_ANNOTATION_DICT),
+        sex_imputation_ploidy_cutoffs=meta_ht.sex_imputation_ploidy_cutoffs,
+        population_inference_pca_metrics=hl.struct(
+            n_pcs=meta_ht.population_inference_pca_metrics.n_pcs,
+            min_prob=meta_ht.population_inference_pca_metrics.min_prob,
+        ),
+        hard_filter_cutoffs=meta_ht.hard_filter_cutoffs,
+    )
+
+    relatedness_ht = relatedness.ht()
+    subset_samples = meta_ht.s.collect()
+    relatedness_ht = relatedness_ht.filter(
+        subset_samples.contains(relatedness_ht.i.s)
+        & subset_samples.contains(relatedness_ht.j.s)
+    )
+
+    relatedness_ht = get_relatedness_set_ht(relatedness_ht)
+    meta_ht = meta_ht.select(
+        bam_metrics=meta_ht.bam_metrics,
+        subsets=meta_ht.subsets.select("tgp", "hgdp"),
+        sex_imputation=meta_ht.sex_imputation.drop("is_female"),
+        sample_qc=meta_ht.sample_qc.select(*SAMPLE_QC_METRICS),
+        relatedness_inference=relatedness_ht[meta_ht.key],
+        population_inference=meta_ht.population_inference.drop(
+            "training_pop", "training_pop_all"
+        ),
+        labeled_subpop=meta_ht.project_meta.project_subpop,
+        gnomad_release=meta_ht.release,
+        high_quality=meta_ht.high_quality,
+    )
+
+    return meta_ht
+
+
+# TODO: move to common area for use in release HT code? Current version is a little different than Grace's version
+def prepare_variant_annotations(ht: hl.Table, filter_lowqual: bool = True) -> hl.Table:
+    """
+    Load and join all Tables with variant annotations.
+
+    :param Table ht: Input HT to add variant annotations to.
+    :param bool filter_lowqual: If true the returned Table will filter out lowqual variants using info HT AS_lowqual.
+    :return: Table containing joined annotations.
+    :rtype: hl.Table
+    """
+    logger.info("Loading annotation tables...")
+    filters_ht = final_filter.ht()
+    vep_ht = vep.ht()
+    dbsnp_ht = dbsnp.ht().select("rsid")
+    info_ht = get_info().ht()
+    analyst_ht = analyst_annotations.ht()
+    freq_ht = get_freq().ht()
+    score_name = hl.eval(filters_ht.filtering_model.score_name)
+
+    if filter_lowqual:
+        logger.info("Filtering lowqual variants...")
+        ht = ht.filter(info_ht[ht.key].AS_lowqual, keep=False)
+
+    logger.info("Assembling 'info' field...")
+    info_fields = SITE_FIELDS + AS_FIELDS
+    missing_info_fields = set(info_fields).difference(info_ht.info.keys())
+    select_info_fields = set(info_fields).intersection(info_ht.info.keys())
+    logger.info(
+        f"The following fields are not found in the info HT: {missing_info_fields}"
+    )
+
+    # NOTE: SOR and AS_SOR annotations are now added to the info HT by default with get_as_info_expr and
+    # get_site_info_expr in gnomad_methods, but they were not for v3 or v3.1
```

Comment:
yes exactly, we used to get these annotations from VQSR, so they were not part of the `get_as_info_expr` and `get_site_info_expr` calculations at the time (they are now there) so the v3.1 SOR has to come from a different table than the INFO HT. Moving forward they will be on our info HT because I added this code to `gnomad_methods`